### PR TITLE
Adds Moonshadow as Westend parachain

### PR DESCRIPTION
**Pull request by crystalin**
Adds Moonshadow in the list of Westend parachain.
Please do not merge until moonbeam types package 1.1.16 is published (Monday morning CET)
Chain should go live soon.

### CODE REVIEW

File: packages/apps-config/src/settings/ethereumChains.ts [hunk 1]
```diff
@@ -8,5 +8,5 @@ export const ethereumChains = [
   'moonbase-alphanet',
   'moonbeam-alphanet',
   'moonbeam-standalone',
-  'moonbeam'
```
Comment (jacogr) [inline packages/apps-config/src/settings/ethereumChains.ts:11]: Not sure we want to drop `moonbeam` here? (I guess it is still valid for some chains?)

Comment (crystalin) [reply packages/apps-config/src/settings/ethereumChains.ts:11]: Thank you

**Comment by crystalin** [comment]
This is ready now @jacogr

File: packages/apps-config/src/endpoints/testingRelayWestend.ts [hunk 2]
```diff
@@ -38,7 +38,15 @@ export function createWestend (t: TFunction): EndpointOption {
...
-      }
+      },
+      {
+        info: 'moonshadow',
+        paraId: 2002,
+        text: t('rpc.westend.moonshadow', 'Moonshadow', { ns: 'apps-config' }),
+        providers: {
+          Purestake: 'wss://wss.moonshadow.testnet.moonbeam.network'
+        }
+      },
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRelayWestend.ts:49]:
```suggestion
      }
```

File: packages/apps-config/src/settings/ethereumChains.ts [hunk 3]
```diff
@@ -8,5 +8,6 @@ export const ethereumChains = [
...
+  'moonbeam',
+  'moonshadow',
```
Comment (jacogr) [inline packages/apps-config/src/settings/ethereumChains.ts:12]:
```suggestion
  'moonshadow'
```

File: packages/apps-config/src/endpoints/testingRelayWestend.ts (hunk 2 above)
Comment (jacogr) [reply packages/apps-config/src/endpoints/testingRelayWestend.ts:49]: This and the one below is what CI is complaining about.

**Comment by jacogr** [comment]
Just run `yarn lint` locally, it should sort out the issues CI is reporting with autofix.

**Review by jacogr — APPROVED** [review]
Excellent. Thank you very much.

**Pull request merged**